src/main/java/Competition.java

```diff
@@ -1,14 +1,17 @@
 public class Competition {
     public static void main(String[] args) {
-
         InputView inputView = new InputView();
 
-        RacingCarCompetition competition = new RacingCarCompetition();
-        competition.numberOfCar = inputView.inputNumberOfCar();
-        competition.numberOfTimes = inputView.inputNumberOfTimes();
-        competition.readyForCompetition();
+        String[] names = inputView.inputNameOfCars();
+        int times = inputView.inputNumberOfTimes();
+
+        RacingCarCompetition competition = new RacingCarCompetition(names);
+        competition.ready(times);
+
+        ResultView resultView = new ResultView(competition.history);
```

Comment:
```suggestion
        ResultView resultView = new ResultView(competition.history());
```
내부 상태/구현을 숨길수록(캡슐화) 유연하고 결합도가 낮은 코드로 만들수 있어요 :)